assets/express/src/api/user/v1/index.ts

```diff
@@ -0,0 +1,24 @@
+import { ControllerFactory } from "@common";
+import { UserService } from "@services";
+import { MiddlewareCollection, UserController } from "../interfaces";
+
+import { getUserRouteOptions } from "./get-user";
+import { createUserRouteOptions } from './create-user'
+import { updateUserRouteOptions } from "./update-user";
+import { deleteUserRouteOptions } from "./delete-user";
+
+import { verifyJWT } from "@middleware";
+
+export const userControllerFactory: ControllerFactory<
```

Comment:
I see this a little differently. The `v1` folder should just export all the routes and this ControllerFactory function should be generic for all the versions and modules.